src/ragas/metrics/_answer_correctness.py

```diff
@@ -159,6 +161,7 @@ class AnswerCorrectness(MetricWithLLM, MetricWithEmbeddings):
     )
     weights: list[float] = field(default_factory=lambda: [0.75, 0.25])
     answer_similarity: AnswerSimilarity | None = None
+    sentence_segmenter: t.Optional[HasSegmentMethod] = None
```

Comment:
```suggestion
    answer_similarity: t.Optional[AnswerSimilarity] = None
    sentence_segmenter: t.Optional[HasSegmentMethod] = None
```

just to keep types consistent through out codebase